backend-rust/src/graphql_api.rs

```diff
@@ -3579,7 +3606,6 @@ struct Delegation {
     staked_amount:     Amount,
     restake_earnings:  bool,
     delegation_target: DelegationTarget,
-    pending_change:    PendingDelegationChange,
```

Comment:
I should probably leave this for backwards compatility and just provide it with some placeholder value

Reply:
This issue has been fixed by not querying it on the frontend:

https://github.com/Concordium/concordium-scan/pull/326